modules/reliability/php/NDB_Menu_Filter_reliability.class.inc

```diff
@@ -69,32 +69,41 @@ class NDB_Menu_Filter_reliability extends NDB_Menu_Filter
         foreach($pscRows AS $row) {
             $this->centerIDMap[$row['CenterID']] = $row['Name'];
         }
-        
+
         $this->_populate_reliability_table();
 
-        $query = " FROM candidate, session, flag, reliability WHERE reliability.CommentID=flag.CommentID AND candidate.CandID=session.CandID AND session.ID=flag.SessionID";
+        $query = " FROM session, flag, " .
+            "reliability LEFT JOIN psc p ON reliability.reliability_center_id=p.CenterID, " .
```

Comment:
Comma is at the end of the line because there is another table with another join following. So it goes like

`SELECT .... FROM table1, table2, (table3 JOIN ... ON ....), (table4 JOIN ... ON ....) WHERE ....`